DemoApp/DemoApp/Views/PopupSamples/SamplePopupPage.xaml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:constants="clr-namespace:DemoApp.Constant"
+             xmlns:Modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             x:Class="DemoApp.Views.SamplePopupPage"
+             Title="SamplePopupPage">
+  <AbsoluteLayout>
+    <Modus:TMButton Clicked="ArrowDownClicked"
```

Comment:
Can you please try with click event with Viewmodel

Reply:
Done